2025-05-27 22:54:12.768 | INFO     | app.infrastructure.llm.qwen_client:<module>:21 - 成功导入dashscope库
2025-05-27 22:54:12.768 | INFO     | app.service.dialogue_context_service:__init__:100 - 对话上下文管理服务初始化成功
2025-05-27 22:54:12.842 | INFO     | app.infrastructure.llm.qwen_client:__init__:58 - 千问大模型客户端初始化完成，使用模型: qwen-max
2025-05-27 22:54:12.842 | INFO     | app.service.llm_service:__init__:23 - 大模型服务初始化成功
2025-05-27 22:54:12.898 | INFO     | app.infrastructure.repository.postgres_repository:__init__:55 - PostgreSQL意图仓储初始化完成
2025-05-27 22:54:12.898 | INFO     | app.service.intent_service:__init__:32 - 意图识别服务初始化成功
2025-05-27 22:54:27.352 | INFO     | app.service.intent_service:recognize_intent:54 - 开始处理意图识别请求，文本: 介绍一下你自己？, 会话ID: user-7ecodd3i
2025-05-27 22:54:27.353 | INFO     | app.service.dialogue_context_service:get_context:116 - 为会话 user-7ecodd3i 创建新的对话上下文
2025-05-27 22:54:27.358 | ERROR    | app.service.llm_service:recognize_intent:66 - 意图识别失败: 'LLMService' object has no attribute 'default_llm_type'
2025-05-27 22:54:27.358 | ERROR    | app.service.intent_service:recognize_intent:147 - 意图识别处理失败: 意图识别失败: 'LLMService' object has no attribute 'default_llm_type'
2025-05-27 22:54:27.358 | ERROR    | app.controller.intent_controller:recognize_intent:81 - 处理意图识别请求失败: 意图识别处理失败: 意图识别失败: 'LLMService' object has no attribute 'default_llm_type'
